### PR TITLE
{chem}[intel/2022a] OpenMolcas v22.10 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

Depends on: ~~#16061~~

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/72353b967cb8c1b6a5acb3f76dac6ccc for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in total)
node075.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/9da616dbb8df3edf9482544aaa428ccf for a full test report.

**Comment by akesandgren** [comment]
Going yto downgrade GA to 5.8.1 since there is already another PR that depends on GA/5.8.1 for this toolchain version.

**Comment by akesandgren** [comment]
@boegelbot Please test @ generoso

File: easybuild/easyconfigs/o/OpenMolcas/OpenMolcas-22.10-intel-2022a-noGA.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+dependencies = [
+    ('Python', '3.10.4'),
+    ('HDF5', '1.12.2'),
+]
+
+build_shared_libs = True
+
+configopts = '-DLINALG=MKL -DMKLROOT=$MKLROOT '
+configopts += '-DOPENMP=ON -DTOOLS=ON -DFDE=ON'
+configopts += '-DMPI=OFF -DGA=OFF '
```
Comment (branfosj) [inline easybuild/easyconfigs/o/OpenMolcas/OpenMolcas-22.10-intel-2022a-noGA.eb:32]: There is a space missing at the end of the second line, so we get `-DFDE=ON-DMPI=ON`. My suggestion to make sure we avoid this issue:
```py
_copts = [
    "-DLINALG=MKL",
    "-DMKLROOT=$MKLROOT",
    "-DOPENMP=ON",
    "-DTOOLS=ON",
    "-DFDE=ON",
    "-DMPI=OFF",
    "-DGA=OFF",
]
configopts = ' '.join(_copts)
```

File: easybuild/easyconfigs/o/OpenMolcas/OpenMolcas-22.10-intel-2022a.eb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    ('HDF5', '1.12.2'),
+    ('GlobalArrays', '5.8.1'),
+]
+
+build_shared_libs = True
+
+preconfigopts = 'GAROOT=$EBROOTGLOBALARRAYS'
+configopts = '-DLINALG=MKL -DMKLROOT=$MKLROOT '
+configopts += '-DOPENMP=ON -DTOOLS=ON -DFDE=ON'
+configopts += '-DMPI=ON -DGA=ON '
```
Comment (branfosj) [inline easybuild/easyconfigs/o/OpenMolcas/OpenMolcas-22.10-intel-2022a.eb:33]: And the same issue here with the lack of a space at the end of the second line.

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/o/OpenMolcas/OpenMolcas-22.10-intel-2022a.eb (hunk 2 above)
Comment (akesandgren) [reply easybuild/easyconfigs/o/OpenMolcas/OpenMolcas-22.10-intel-2022a.eb]: Fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/16770/commits/41bf0895e03a05391cbd1510136f6a9cb213ff36

**Comment by akesandgren** [comment]
@boegelbot Please test @ jsc-zen2

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/385acd59978418419df9db6d714fc22d for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bear-pg0105u36b.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/68280e0d2cdc8a86f4957c0500d529b5 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @akesandgren!

**Pull request merged**